### PR TITLE
Fix error thrown for nullable field resolving null

### LINKED ISSUE #22: Error thrown when resolving null for a nullable object type
When a field is an object type that is not non-nullable, resolving it to `null` throws an error, while it's expected to just resolve `null`.

```js
const typeDefs = `
  type Query {
    foo: Foo
  }
  type Foo {
    bar: Int
  }
`;

const resolvers = {
  Query: {
    foo: () => {
      return null;
    }
  }
};


const query = gql`
  query {
    foo {
      bar
    }
  }
`;

// reactive-graphql
$graphql(schema, query)
// -> error reactive-graphql: resolver emitted empty value 

// reference graphql-js
execute(schema, query)
// -> {"data":{"foo":null}} 
```

https://codesandbox.io/s/4qvw8y31yw

### PR DESCRIPTION
**Pull request by alexstrat**
Closes #22

### CODE REVIEW

File: src/__tests__/graphqlObservable-test.ts [hunk 1]
```diff
@@ -488,6 +492,29 @@ describe("graphqlObservable", function() {
           m.expect(result.pipe(take(1))).toBeObservable(expected);
         });
 
+        itMarbles.only("if nullable field resolver returns null, it resolves null", function(m) {
```
Comment (alexstrat) [inline src/__tests__/graphqlObservable-test.ts]: ⚠️

Comment (alexstrat) [reply src/__tests__/graphqlObservable-test.ts]: #24

**Comment by DanielMSchmidt** [comment]
Seems cool, but I think the commits are not conform to [semantic release](https://github.com/semantic-release/semantic-release), so they won't trigger a release. I would count this as a simple fix, so there could be a single commit like this `fix: error thrown for nullable field resolving null`

**Comment by alexstrat** [comment]
@DanielMSchmidt ok, can't you [squash while merging](https://help.github.com/en/articles/merging-a-pull-request) and rename as sematic wishes?

**Comment by DanielMSchmidt** [comment]
Done 👍

**Pull request merged**